cmd/subnetcmd/deploy.go

```diff
@@ -171,7 +172,7 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	var threshold uint32
 
 	if len(controlKeys) > 0 {
-		threshold, err = getThreshold(uint64(len(controlKeys)))
+		threshold, err = getThreshold(len(controlKeys))
```

Comment:
I am not completelly sure that len(controlKeys) should be the max. I think a max of somewhat 3 was refered to in some meeting.

Reply:
I don't know of any such restriction, can you find the source please?

Reply:
Resolved as per https://github.com/ava-labs/avalanche-cli/pull/181#issuecomment-1216984871